server/run-tests.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+LOG_LEVEL="test"
+
+while [[ "$#" -gt 0 ]]; do
+  case $1 in
+  --debug)
+    LOG_LEVEL="debug"
+    shift
+    ;;
+  esac
+  shift
+done
+
+export NODE_ENV="test"
+export SYNC_DATABASE="false"
+export SQL_DATABASE="dds_test"
+export LOG_LEVEL="$LOG_LEVEL"
+
+# Create test database if it doesn't exist.
+if [ "$(psql -tAc "SELECT 1 FROM pg_database WHERE datname='$SQL_DATABASE'" -h "$SQL_HOST" -p "$SQL_PORT" -U "$SQL_USER")" != '1' ]; then
+  echo -n "Creating '$SQL_DATABASE' database... "
+  createdb -h "$SQL_HOST" -p "$SQL_PORT" -U "$SQL_USER" "$SQL_DATABASE" || {
+    echo "failed to create '$SQL_DATABASE' database. Aborting..."
+    exit
+  }
+  echo "success!"
+fi
+
+../migration-run.sh
+npx --silent ts-mocha \
```

Comment:
The place of `--silent` applied to this command caused an error when trying to run the first time. Placing it before the command caused this error to throw:
`npx --silent ts-mocha   --project "tsconfig.json"   --config "./.mocharc.yml"   --exit   "./**/*.spec.ts"
 ERROR: You must supply a command.
`
Placing after the `ts-mocha` command seems to fix this issues